force-app/main/default/flows/HOT_Request_Initiator.flow-meta.xml

```diff
@@ -0,0 +1,126 @@
+        <locationX>176</locationX>
+        <locationY>575</locationY>
+        <actionName>customNotificationAction</actionName>
+        <actionType>customNotificationAction</actionType>
+        <flowTransactionModel>CurrentTransaction</flowTransactionModel>
+        <inputParameters>
+            <name>customNotifTypeId</name>
+            <value>
+                <elementReference>HOT_Get_Request_Created_Id.Id</elementReference>
+            </value>
+        </inputParameters>
+        <inputParameters>
+            <name>body</name>
+            <value>
+                <stringValue
+                >Det har blitt opprettet en Forespørsel i HOT Tolk Admin køen i produksjon. Se detaljer her: {![HOT_Request__c].Name}.</stringValue>
+            </value>
+        </inputParameters>
+        <inputParameters>
+            <name>title</name>
+            <value>
+                <stringValue>Opprettet en Forespørsel i HOT Tolk Admin køen</stringValue>
+            </value>
+        </inputParameters>
+        <inputParameters>
+            <name>recipientIds</name>
+            <value>
+                <elementReference>RecipientIDs</elementReference>
+            </value>
+        </inputParameters>
+        <inputParameters>
+            <name>targetId</name>
+            <value>
+                <elementReference>$Record.Id</elementReference>
+            </value>
+        </inputParameters>
+    </actionCalls>
+    <apiVersion>56.0</apiVersion>
+    <assignments>
+        <name>HOT_Add_Owner_to_the_RecipientIDs</name>
+        <label>Add Owner to the RecipientIDs</label>
+        <locationX>176</locationX>
+        <locationY>455</locationY>
+        <assignmentItems>
+            <assignToReference>RecipientIDs</assignToReference>
+            <operator>Add</operator>
+            <value>
+                <elementReference>$Record.OwnerId</elementReference>
+            </value>
+        </assignmentItems>
+        <connector>
+            <targetReference>HOT_Create_Notification</targetReference>
+        </connector>
+    </assignments>
+    <description>Notify Owner if there has been created a request in the HOT_Tolk_Admin queue.</description>
+    <environments>Default</environments>
+    <interviewLabel>Request Initiator {!$Flow.CurrentDateTime}</interviewLabel>
+    <label>Request Initiator</label>
+    <processMetadataValues>
+        <name>BuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>CanvasMode</name>
+        <value>
+            <stringValue>AUTO_LAYOUT_CANVAS</stringValue>
+        </value>
+    </processMetadataValues>
+    <processMetadataValues>
+        <name>OriginBuilderType</name>
+        <value>
+            <stringValue>LightningFlowBuilder</stringValue>
+        </value>
+    </processMetadataValues>
+    <processType>AutoLaunchedFlow</processType>
+    <recordLookups>
```

Comment:
Henter ut id til CustomNotificationType? I så fall bør du endre navnet. Skjønner ikke helt spørringen heller for du filtrerer på Id = Record.Id, bør du ikke her hente ut CustomNotificationType basert på developerName?